java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/sdk/impl/SqlClusterExecutor.java

```diff
@@ -62,27 +62,28 @@ public class SqlClusterExecutor implements SqlExecutor {
     private static final AtomicBoolean initialized = new AtomicBoolean(false);
     private SQLRouter sqlRouter;
     private DeploymentManager deploymentManager;
-    private ZKClient zkClient;
     private InsertPreparedStatementCache insertCache;
 
     public SqlClusterExecutor(SdkOption option, String libraryPath) throws SqlException {
         initJavaSdkLibrary(libraryPath);
-
+        ZKClient zkClient = null;
         if (option.isClusterMode()) {
             SQLRouterOptions sqlOpt = option.buildSQLRouterOptions();
             this.sqlRouter = sql_router_sdk.NewClusterSQLRouter(sqlOpt);
             sqlOpt.delete();
-            zkClient = new ZKClient(ZKConfig.builder()
-                    .cluster(option.getZkCluster())
-                    .namespace(option.getZkPath())
-                    .sessionTimeout((int)option.getSessionTimeout())
-                    .build());
-            try {
-                if (!zkClient.connect()) {
-                    throw new SqlException("zk client connect failed.");
+            if (!option.isLight()) {
+                zkClient = new ZKClient(ZKConfig.builder()
+                        .cluster(option.getZkCluster())
+                        .namespace(option.getZkPath())
+                        .sessionTimeout((int)option.getSessionTimeout())
+                        .build());
+                try {
+                    if (!zkClient.connect()) {
+                        throw new SqlException("zk client connect failed.");
+                    }
+                } catch (Exception e) {
+                    throw new SqlException("init zk client failed. " + e.getMessage());
```

Comment:
It's better to use `throw new xxExecption("xxx",e)`, so we can get the trace. RuntimeExecption or improve the SqlException?